src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -772,44 +771,49 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
             /// and should show up after fixes but before suppression fixes in the light bulb menu.
             /// </remarks>
             private SuggestedActionSet OrganizeRefactorings(
-                Workspace workspace, CodeRefactoring refactoring, Span applicableSpan)
+                Workspace workspace, CodeRefactoring refactoring)
             {
                 var refactoringSuggestedActions = ArrayBuilder<SuggestedAction>.GetInstance();
 
-                foreach (var action in refactoring.Actions)
+                foreach (var codeAction in refactoring.CodeActions)
                 {
-                    if (action.NestedCodeActions.Length > 0)
+                    if (codeAction.action.NestedCodeActions.Length > 0)
                     {
-                        var nestedActions = action.NestedCodeActions.SelectAsArray(
+                        var nestedActions = codeAction.action.NestedCodeActions.SelectAsArray(
                             na => new CodeRefactoringSuggestedAction(
                                 ThreadingContext,
                                 _owner, workspace, _subjectBuffer, refactoring.Provider, na));
 
                         var set = new SuggestedActionSet(categoryName: null,
-                            actions: nestedActions, priority: GetSuggestedActionSetPriority(action.Priority), applicableToSpan: applicableSpan);
+                            actions: nestedActions, priority: GetSuggestedActionSetPriority(codeAction.action.Priority), applicableToSpan: codeAction.applicableToSpan?.ToSpan());
```

Comment:
when can the span be null?

Reply:
When the legacy overload that didn't specify applicableToSpan is used. We could've gone with providing either `default` or user span but the first option is just wrong and the second one makes all "legacy" refactorings automatically the closest. And that also didn't feel right.

There's a whole discussion about that on the PR this one is based on. #37777